proto/api/v1/compliance_service.proto

```diff
@@ -102,12 +98,6 @@ service ComplianceService {
         };
     }
 
-    rpc GetComplianceControlResults(RawQuery) returns (ComplianceControlResultsResponse) {
-        option (google.api.http) = {
-            get: "/v1/compliance/results"
```

Comment:
Will add a changelog entry for this, but this does not need to be deprecated as it was never implemented so no customers are relying on it